src/code/templates/fv-egg-game.js

```diff
@@ -1095,7 +1094,7 @@ export default class FVEggGame extends Component {
                                   selectedChromosomes: fatherSelectedChromosomes };
       return <GenomeView className={parentGenomeClass}  {...uniqueProps} ChromosomeImageClass={FVChromosomeImageView}
                         small={ true } editable={false} userChangeableGenes={ userChangeableGenes } visibleGenes={ visibleGenes }
-                        onAlleleChange={ handleAlleleChange }
+                        onAlleleChange={ handleAlleleChange } isSelectedEmpty={true} chromosomeHeight={122}
```

Comment:
In the particular case of `FVEggGame`, the clients are `parentGenomeView()` which wants to show _unselected_ chromosomes as filled, while `parentHalfGenomeView()` wants to show _selected_ chromosomes as filled. So for example, `parentGenomeView()` would pass `selectedChromosomes: invertChromosomes(motherSelectedChromosomes)`. There's an argument for naming the prop `filledChromosomes` or something like that, rather than `selectedChromosomes` while in the area.